problems/objects.js

```diff
@@ -187,16 +274,51 @@ const countOccurance = () => {};
  * @returns {Object} Counts all characters except spaces  {a: 2, g: 1, o: 2, d:1, s: 1, n:a, k:1, e: 1}
  */
 
-const countOccuranceNoSpaces = () => {};
+const countOccuranceNoSpaces = (str) => {
+  let lilStr = str.toLowerCase()
+  let countObj = {}
+  for(let i=0;i<lilStr.length;i++){
+    let el = lilStr[i]
+    if(countObj[el]){
+      if(countObj[el]!==" "){
+        countObj[el]+=1}
+    }else{
+      if(el !== " ") {
+        countObj[el] = 1}
+    }
+  }
+  return countObj
+};
 
 /**
  * Takes in an array and returns the most common element.
  * @param {Array} array - array of elements
  * @returns {(number|string)} Most common element
  */
 
-const mostCommonElement = () => {};
-
+const mostCommonElement = (array) => {
+  let obj = {}
+  for (let i = 0; i < array.length; i++) {
+    let element = array[i]
+    if (obj[element]) {
+        obj[element] += 1
+    } else {
+      obj[element] = 1
+    }
+    } let mostCommon = -Infinity
```

Comment:
Please be mindful of syntax. Keep an eye on tabbing and curly braces, or use prettier in VSCode.